selfdrive/car/ford/values.py

```diff
@@ -195,6 +195,7 @@ def init_make(self, CP: car.CarParams):
       b'MB5A-14C204-RC\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00',
       b'NB5A-14C204-AZD\x00\x00\x00\x00\x00\x00\x00\x00\x00',
       b'NB5A-14C204-HB\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00',
+      b'A714B7D487084AB7B15\x00\x00\x00\x00\x00',
```

Comment:
@incognitojam know why the FW looks so different? It's a Floridian 2021 Explorer

Reply:
Oh, is it the real camera? https://www.ebay.com/itm/203267707228

Reply:
I don't think that's the LKAS camera. If it's park assist and "camera" it's probably to do with the IPMB or reversing camera..

Reply:
It could be caused by an engine tune.

I did do a full scan for ECUs, and there are more things like the gateway module and infotainment system. My car doesn't have a lot of extras like a reversing camera so I wouldn't have seen that :/

Reply:
do the extra_ecus get queried _after_ fingerprinting is complete? if they do then it's worth adding them I suppose

Reply:
No, they're queried at the same time.

Reply:
@incognitojam did you hear anything back from the user on Discord?

Reply:
nothing beyond our initial conversation

Reply:
User reports that it has an engine tune

Reply:
The user used to have this engine FW: `b'MB5A-14C204-MD\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'`

Maybe we just add this and ask the user to revert their tune to use openpilot

Reply:
oh, it's already in there!